src/components/home/Search.tsx

```diff
@@ -0,0 +1,191 @@
+import Button from '../common/Button';
+import SearchBottom from './SearchBottom';
+import SearchCheckedBox from './SearchCheckedBox';
+import SearchClickedFieldWrapper from './SearchClickedFieldWrapper';
+import SearchRadioButton from './SearchRadioButton';
+import SearchTextField from './SearchTextField';
+import { CalendarGreyIcon, OriginIcon, PinIcon, ProfileIcon } from '@/assets/svg';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+const Search = () => {
+	const [selectedRadio, setSelectedRadio] = useState<string>('왕복');
+	const handleRadioClick = (value: string) => {
+		setSelectedRadio(value);
+	};
+
+	const [selectedNearbyDeparture, setSelectedNearbyDeparture] = useState(false);
+	const [selectedNearbyArrival, setSelectedNearbyArrival] = useState(false);
+	const [selectedDirect, setSelectedDirect] = useState(false);
```

Comment:
p4: 해당 부분은 체크박스 상태이므로, selected보다는 checked로 수정하여 직관적으로 해당 상태를 이해할 수 있게 수정하는 것이 어떨까요?
```suggestion
	const [checkedNearbyDeparture, setCheckedNearbyDeparture] = useState(false);
	const [checkedNearbyArrival, setCheckedNearbyArrival] = useState(false);
	const [checkedDirect, setCheckedDirect] = useState(false);
```